src/nl/hannahsten/texifyidea/TexifyIcons.kt

```diff
@@ -469,29 +231,27 @@ object TexifyIcons {
      *
      * @param extension
      *              The extension of the file to get the icon of without a dot.
-     * @param smaller
-     *              `true` for a small icon, `false` for a regular sized icon.
      * @return The Icon that corresponds to the given extension.
      * @throws IllegalArgumentException
      * When `extension` is null.
      */
-    fun getIconFromExtension(extension: String?, smaller: Boolean = false): Icon {
+    fun getIconFromExtension(extension: String?): Icon {
         return if (extension == null) {
             FILE
         }
         else when (extension.lowercase(Locale.getDefault())) {
-            "tex" -> if (smaller) LATEX_FILE_SMALLER else LATEX_FILE
-            "bib" -> if (smaller) BIBLIOGRAPHY_FILE_SMALLER else BIBLIOGRAPHY_FILE
-            "cls" -> if (smaller) CLASS_FILE_SMALLER else CLASS_FILE
-            "dtx" -> if (smaller) DOCUMENTED_LATEX_SOURCE_SMALLER else DOCUMENTED_LATEX_SOURCE
-            "sty" -> if (smaller) STYLE_FILE_SMALLER else STYLE_FILE
-            "txt" -> if (smaller) TEXT_FILE_SMALLER else TEXT_FILE
-            "tikz" -> if (smaller) TIKZ_FILE_SMALLER else TIKZ_FILE
-            "log" -> if (smaller) TEXT_FILE_SMALLER else TEXT_FILE
-            "pdf" -> if (smaller) PDF_FILE_SMALLER else PDF_FILE
-            "synctex.gz" -> if (smaller) SYNCTEX_FILE_SMALLER else SYNCTEX_FILE
-            "dvi" -> if (smaller) DVI_FILE_SMALLER else DVI_FILE
-            else -> if (smaller) FILE_SMALLER else FILE
+            "tex" -> LATEX_FILE
+            "bib" -> BIBLIOGRAPHY_FILE
+            "cls" -> CLASS_FILE
+            "dtx" -> DOCUMENTED_LATEX_SOURCE
+            "sty" -> STYLE_FILE
+            "txt" -> TEXT_FILE
+            "tikz" -> TIKZ_FILE
+            "log" -> TEXT_FILE
+            "pdf" -> PDF_FILE
+            "synctex.gz" -> SYNCTEX_FILE
+            "dvi" -> DVI_FILE
```

Comment:
Would it make sense to reuse the hardcoded list from IconMagic here? It seems to mostly overlap

Reply:
I'd even go so far as to remove  IconMagic and move it all to `TexifyIcons#getIconFromExtension`.

Reply:
Agree